qutebrowser/config/configdata.py

```diff
@@ -766,6 +766,11 @@ def data(readonly=False):
              "required to exactly match the requested domain.\n\n"
              "Local domains are always exempt from hostblocking."),
 
+            ('enable-pdfjs', SettingValue(typ.Bool(), 'true'),
```

Comment:
I'd prefer if this was `false` by default - I think many people prefer to opt-in to pdfjs than to opt-out of it :wink: